### ISSUE TITLE
Losing control over the cameras on a dodgy connection

### ISSUE THREAD

**Issue by arnthorg** [AdamSanderson93/PyGigEV#4]
Hi!

I'm testing out the code with cameras over a wi-fi bridge which can sometimes be dodgy. 

After some time I lose all control over the cameras and whatever command I try (except GevOpenCamera*) I get -4, invalid handle. and when I try to open the cameras I either get x8006 access denied or  x8fff generic error. GevCloseCam also returns -4, invalid handle.  

The cameras are still running at that time and sending frames over but I can't get control over them again nor get the frames from them. So it's like the cameras don't time out and are being kept alive by some routine. I was wondering if maybe the camera handles get corrupted or s-t. 

When I restart the program they work fine again so I could just exit the program when that happens and have a watchdog rerun the program when it's exited, do you know if there's a more elegant way?  I couldn't unload the pygigev module. 

I'm seeing if I can get Gev_Reconnect() to work, 
edit: Gev_Reconnect() also returns -4, invalid handle

**Comment by AdamSanderson93** [comment]
@arnthorg I believe this issue is from GigEV. I ran into a similar problem with this library in C++. I had the connection drop and I tried everything from deleting the camera objects, and closing the connections to even unloading the library. The only thing I could get to work was to sigkill the entire node. Have it crash and relaunch it.... I definitely wouldn't recommend using this over a wifi bridge. If possible have a local PC connect to the camera and control it then use GStreamer or a different streaming library to forward the footage.
